### PR TITLE
Update device-health-api-methods-properties.md

### PR DESCRIPTION
**Pull request by msbemba**
added rate limitations to the document

### CODE REVIEW

**Comment by msbemba** [comment]
per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10123

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/device-health-api-methods-properties.md [hunk 1]
```diff
@@ -66,7 +66,12 @@ Method|Data type|Description
...
 **(via files)**|Microsoft Defender Antivirus health per device collection. See: [1.3 Export device antivirus health details API properties \(via files\)](#13-export-device-antivirus-health-details-api-properties-via-files)|Returns a table with an entry for every unique combination of DeviceId, ConfigurationId. |This API solution enables pulling larger amounts of data faster and more reliably. So, it's recommended for large organizations, with more than 100-K devices. This API pulls all data in your organization as download files. The response contains URLs to download all the data from Azure Storage. This API enables you to download all your data from Azure Storage as follows: <ol><li>Call the API to get a list of download URLs with all your organization data.</li><li>Download all the files using the download URLs and process the data as you like.</li></ol>
 
-### 1.2 Export device antivirus health details API properties (JSON response)
+#### 1.2 Limitations
+
+- maximum page size is 200,000
+- Rate limitations for this API are 30 calls per minute and 1000 calls per hour.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-health-api-methods-properties.md]:
```suggestion
### 1.2 Limitations

- Maximum page size: 200,000
- Rate limitations for this API: 30 calls per minute and 1,000 calls per hour
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**